services/gitea/9.3.0/pvc.yaml

```diff
@@ -0,0 +1,22 @@
+---
+# In the previous versions (v8.x and prior), PVC was automatically managed by the Statefulset, while in the
+# new versions (v9.x and later), it is automatically handled by the Helm chart.
+#
+# To avoid breaking the compatibility with previous installations, we do not let Helm chart create PVC (see `persistence.create` in cm.yaml file)
+# and for new installations we create a PVC object using the name `data-gitea-0` that is the same name used in
+# gitea v8.x and older versions. This way, we can keep the existing PVC for upgrade process.
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: data-gitea-0
+  namespace: ${releaseNamespace}
+  annotations:
+    helm.sh/resource-policy: keep
```

Comment:
now that we maintain this PVC behind a Flux Kustomization that has `prune: true`, I was thinking it might be worth adding this annotation to prevent pruning (it might help us in the future)

`kustomize.toolkit.fluxcd.io/prune: disabled`

https://fluxcd.io/flux/components/kustomize/kustomizations/#kustomizetoolkitfluxcdioprune

Reply:
very good idea!